src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -325,6 +319,50 @@ await supersededPublications
                 .OnSuccess(publication => _mapper.Map<PublicationViewModel>(publication));
         }
 
+        public async Task<Either<ActionResult, ExternalMethodologyViewModel>> GetExternalMethodology(Guid publicationId)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                .OnSuccessDo(_userService.CheckCanViewPublication)
+                .OnSuccess(publication => publication.ExternalMethodology != null
+                    ? _mapper.Map<ExternalMethodologyViewModel>(publication.ExternalMethodology)
```

Comment:
You could not bother with AutoMapper here and use `new ExternalMethodologyViewModel(publication.ExternalMethodology)` since you've added  a copy constructor to `ExternalMethodologyViewModel` which copies the `ExternalMethodology`.

Reply:
Also added a constructor for `ExternalMethodologyViewModel` that accepts a `ExternalMethodlogySaveRequest`, so I can remove that from the mapping config too.